substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/BuildArtifacts.java

```diff
@@ -47,6 +47,8 @@ enum ArtifactType {
         JDK_LIBRARY("jdk_libraries"),
         /* For all library shims for the JDK needed at run-time. */
         JDK_LIBRARY_SHIM(JDK_LIBRARY.getJsonKey()), // distinction should not be important to users.
+        /* For all configuration files from java.home needed at run-time */
+        JDK_CONFIG_FILE("jdk_config"),
```

Comment:
If you extend this enum, you'll   also have to extend [this JSON schema](https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/build-artifacts-schema-v0.9.0.json). Since `v0.9.0` is already release, you'll have to copy it and call it `v0.9.1`. Also, search for references to the file name and update the help text of the corresponding option. We should probably add a note about this to the enum's javadoc.

Reply:
example build artifacts file:
```json
{
  "executables": [
    "native-tests.exe.exe"
  ],
  "jdk_config": [
    "lib\\fontconfig.bfc"
  ],
  "jdk_libraries": [
    "awt.dll",
    "fontmanager.dll",
    "freetype.dll",
    "javaaccessbridge.dll",
    "javajpeg.dll",
    "jawt.dll",
    "lcms.dll",
    "java.dll",
    "jvm.dll"
  ]
}
```
I'm assuming the double backslash may have to be normalized?